istio-control/istio-autoinject/templates/clusterrole.yaml

```diff
@@ -9,7 +9,9 @@ metadata:
 rules:
 - apiGroups: [""]
   resources: ["configmaps"]
+  resourceNames: ["istio-sidecar-injector"]
   verbs: ["get", "list", "watch"]
 - apiGroups: ["admissionregistration.k8s.io"]
   resources: ["mutatingwebhookconfigurations"]
+  resourceNames: [{{ print "\"istio-sidecar-injector-" .Release.Namespace "\"" }}]
```

Comment:
minor point: I think this can just be `["istio-sidecar-injector-{{.Release.Namespace}"]` or does that not work?

Reply:
good point - worth simplifying this if posible.

Reply:
Thanks @howardjohn. I updated the PR with the suggested change.